index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width", initial-scale=1.0>
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" href="http://fonts.googleapis.com/css?family=Roboto" >
+</head>
+<body>
+    <header>
+        <div>Ashmal Zahra</div>
+        <nav class="menu">
+           <img src="images/Union.png" alt="Menu">
```

Comment:
- [ ] Your hamburger should be a button or an `a` tag. This is because it is a button that when clicked on the mobile version, it displays the nav options.